src/main/java/seedu/address/model/client/Client.java

```diff
@@ -31,12 +31,13 @@
     private final ContractExpiryDate contractExpiryDate;
```

Comment:
Should the equals and hashCode methods be updated for this class?

Reply:
Nope, refer to PR description, tests currently rely on comparing two clients who are created at different timings, and this is pretty much just metadata for us I guess